doc/user_guide/eds.rst

```diff
@@ -10,12 +10,32 @@ The methods described here are specific to the following signals:
 * :py:class:`~._signals.eds_tem.EDSTEMSpectrum`
 * :py:class:`~._signals.eds_sem.EDSSEMSpectrum`
 
-This chapter described step by step a qualitative analysis of an EDS 
+This chapter described step by step the analysis of an EDS 
 spectrum (SEM or TEM). 
 
+.. NOTE::
+	See also the `EDS tutorials <http://nbviewer.ipython.org/github/hyperspy/hyperspy-	demos/blob/master/electron_microscopy/EDS/>`_ .
+
 Spectrum loading and parameters
 -------------------------------
 
+The files needed for this section can be downloaded using
```

Comment:
I would rather say "Data files used in the following examples can be downloaded using". At the moment it looks as if "you HAVE to download these files in order to use these features"